### PR TITLE
Fix/289/sensorconnect always returns values as string

### PR DESCRIPTION
**Pull request by EmlingerM**
fixes: https://github.com/united-manufacturing-hub/united-manufacturing-hub/issues/289

Sensorconnect should return values according to the in the [IODD spec.](https://io-link.com/share/Downloads/Package-2024-released/IOL-Interface-Spec_10002_V114_Jun24.pdf) specified type instead of string

```json
{
   "Distance":"47",
   "Switch state [OUT1]":"1",
   "connected":"connected",
   "timestamp_ms":"1728629682975",
   "type":"Io-Link"
}

```

=>

```json
{
   "Distance":47,
   "Switch state [OUT1]":true,
   "connected":"connected",
   "timestamp_ms":1728631520855,
   "type":"Io-Link"
}
```
Implemented Types:

  ✅  `BoolenT`IOLINK Spec. V1.1.4 F.2.2
  ✅  `UIntegerT`IOLINK Spec. V1.1.4 F.2.3
  ✅  `IntegerT` IOLINK Spec. V1.1.4 F.2.4
  ✅  `Float32T` IOLINK Spec. V1.1.4 F.2.5
  ✅  `OctetStringT` IOLINK Spec.  V1.1.4 F.2.7

Not Implemented Types:

  ❌   `StringT` IOLINK Spec.  V1.1.4 F.2.6
  ❌   `TimeT` IOLINK Spec.  V1.1.4 F.2.8
  ❌   `TimeSpanT` IOLINK Spec.  V1.1.4 F.2.9
  ❌   `ArrayT` IOLINK Spec.  V1.1.4 F.3.2
  ❌   `RecordT` IOLINK Spec.  V1.1.4 F.3.2

### CODE REVIEW

File: golang/cmd/factoryinsight/main.go [hunk 1]
```diff
@@ -371,7 +371,7 @@ func setupRestAPI(accounts gin.Accounts, version int) {
 
 	*/
 	//dataFormat
-	err := router.Run(":80")
+	err := router.Run(":8080")
```
Comment (Scarjit) [inline golang/cmd/factoryinsight/main.go:374]: I think you might have an accidential/test change in here.

**Review by Scarjit — COMMENTED** [review]
Hi @EmlingerM ,

thanks for your PR, it looks good to me.

File: golang/cmd/factoryinsight/main.go (hunk 1 above)
Comment (EmlingerM) [reply golang/cmd/factoryinsight/main.go]: Hi, sorry that was an accident, I removed it from the commits.

**Review by Scarjit — APPROVED** [review]

**Comment by Scarjit** [comment]
Once you signed the CLA (or added your git email to github, if you already signed it), i will merge this to staging.
The build process seems to have some problems with external PR's, but thats not for this pr to solve :)

**Comment by JeremyTheocharis** [comment]
@EmlingerM I just moved the code into benthos-umh, so that it is independent of the CPU architecture, as well as our (very old) MQTT and Kafka implementation. Go check it out here: https://github.com/united-manufacturing-hub/benthos-umh/pull/68

**Pull request merged**